config/rbac/tfjob_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit tfjobs.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tfjob-editor-role
+rules:
+- apiGroups:
+  - kubeflow.org
+  resources:
+  - tfjobs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - kubeflow.org
+  resources:
+  - tfjobs/status
+  verbs:
+  - get
```

Comment:
do we need to add `update`, `patch` to resources `tfjobs/status` as well as mxnetjobs, pytorchjobs and xboostjobs?

Reply:
I remember we use same tool to generate editor roles.  Do they look different now?

Reply:
@zw0610 seems rest of the role are same.  update, patch are not part of the editor role. 

BTW, this is different from the role running the operator. I create a role to use for operator in #1325

Reply:
SGTM